src/cli/utils.js

```diff
@@ -49,3 +49,15 @@ exports.getIPFS = (callback) => {
 exports.getRepoPath = () => {
   return process.env.IPFS_PATH || os.homedir() + '/.ipfs'
 }
+
+exports.createLogger = (visible) => {
+  return (msg, newline = true) => {
```

Comment:
We do provide a compiled version when publishing do npm, but the tests run on node against the original source atm.